docs/Reference.md

```diff
@@ -286,6 +287,14 @@ Timestamps are expressed in seconds and display the given timestamp in the user'
 | F     | Tuesday, 20 April 2021 16:20 | Long Date/Time  |
 | R     | 2 months ago                 | Relative Time   |
 
+###### Id Types
+
+| Types     | Description                                                                                           |
+|-----------|-------------------------------------------------------------------------------------------------------|
+| customize | Linked to the _Customize_ screen of [Onboarding](#DOCS_RESOURCES_GUILD/guild-onboarding-object)       |
+| browse    | Linked to the _Browse Channels_ screen of [Onboarding](#DOCS_RESOURCES_GUILD/guild-onboarding-object) |
+| guide     | Linked to the _Server Guide_ screen of [Onboarding](#DOCS_RESOURCES_GUILD/guild-onboarding-object)    |
+
 \*default
```

Comment:
```suggestion
```
I'd avoid moving the default type annotation away from its corresponding timestamp styles table.

Reply:
Oops ^^'